routes/home.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+
+const router = express.Router();
+const { db } = require('../db-config');
+
+router.get('/', async (req, res) => {
+  try {
+    const [home] = await db.query(`
+    SELECT source, image, description, date
+    FROM news
+  `);
+    res.json(home);
+  } catch (err) {
+    res.status(404).send();
```

Comment:
Why did you keep the send() if it's empty ??

Reply:
i have remove .send